packages/utils/mocks/setup/cliui.mock.ts

```diff
@@ -0,0 +1,13 @@
+import {beforeEach, vi} from "vitest";
+import {ui} from "../../src/lib/logging";
```

Comment:
Don't know why format check didn't catch this:

```suggestion
import { beforeEach, vi } from 'vitest';
import { ui } from '../../src/lib/logging';
```

Reply:
will double check